backend/store/etcd/event_store.go

```diff
@@ -182,7 +182,7 @@ func (s *Store) GetEventByEntityCheck(ctx context.Context, entityName, checkName
 		return nil, err
 	}
 
-	resp, err := s.client.Get(ctx, path, clientv3.WithPrefix())
+	resp, err := s.client.Get(ctx, path, clientv3.WithPrefix(), clientv3.WithSerializable())
```

Comment:
Could we also add this option to `GetEvents` & `GetEventsByEntity`, or there's no real benefit?

Reply:
We could, but those methods aren't used by the event pipeline, so I expect that adding the option there would have less of an effect.